src/scene/data.ts

```diff
@@ -44,6 +47,61 @@ export function saveAsJSON(
   );
 }
 
+export async function exportToShortlink(
+  elements: readonly ExcalidrawElement[]
+) {
+  const serialized = JSON.stringify({
+    version: 1,
+    source: window.location.origin,
+    elements: elements.map(({ shape, ...el }) => el)
+  });
+
```

Comment:
Can you refactor out the logic that creates this so that we don’t have it in two places that are going to diverge?